shell/edit/monitoring.coreos.com.receiver/index.vue

```diff
@@ -148,8 +161,24 @@ export default {
 
     createAddOptions(receiverType) {
       return receiverType.addOptions.map();
+    },
+
+    createSmarthost() {
+      if (this.value.spec.email_configs.length > 0) {
+        this.value.spec.email_configs.forEach((email) => {
+          if (email['port'] || email['host']) {
+            email.smarthost = `${ email.host }:${ email.port }`;
```

Comment:
If port is not specified, you get a colon at the end - we should only add the : and port if port is not empty

Reply:
Fixed